src/app/ui/components/tabs/tabs.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { Tabs } from './tabs';
+
+const meta: Meta<typeof Tabs.Root> = {
+  component: Tabs.Root,
+  tags: ['autodocs'],
+  title: 'Tabs',
+};
+
+export default meta;
+type Story = StoryObj<typeof Tabs.Root>;
+
+export const Component: Story = {
```

Comment:
I think this will just show `Component` now in the storybook UI? Is that what we want?

Reply:
Yeah it will. We were suggesting to call it `Tabs` purely so it will say `Tabs` in the storybook UI

Reply:
got it